examples/catalog/lib/animated_list.dart

```diff
@@ -117,9 +117,9 @@ class ListModel<E> {
     @required this.listKey,
     @required this.removedItemBuilder,
     Iterable<E> initialItems,
-  }) : assert(listKey != null),
-       assert(removedItemBuilder != null),
-       _items = List<E>.from(initialItems ?? <E>[]);
+  })  : assert(listKey != null),
+        assert(removedItemBuilder != null),
+        _items = List<E>.from(initialItems ?? <E>[]);
```

Comment:
i don't really like the two spaces between `)` and `:` but if it was the only change then i could live with it.